hooks/useForm.ts

```diff
@@ -83,13 +84,20 @@ export function useField(provided: ProvidedField): Field | null {
     isRadioed && radioOption ? radioOption : ''
   )
   const [$disabled, $setDisabled] = useState<boolean>(false)
-  const [, $setValid] = useState<boolean>(false)
+  const [$valid, $setValid] = useState<boolean>(initiallyValid)
   const [$touched, $setTouched] = useState<boolean>(touched)
   const [$field, $setField] = useState<Field | null>(null)
   const [$error, $setError] = useState<string>(defaultErrorText)
   useEffect(() => {
-    const valid = !$touched || ($touched && validation($value))
-    $setValid(valid)
+    if (defaultValue) {
```

Comment:
Is this necessary if we are already making the change to make `$valid` true on line 87? Seems like that would be enough to fix the validation error on submit for the original bug. But idk what `$touched` does. Does it influence some other logic?

Reply:
That's correct, the `if (defaultValue) {` portion of the effect isn't needed. It'll have the effect of never validating subsequent changes to `$value` unless `defaultValue` is unset.

Reply:
I sliced out another `useEffect`, as simply removing the `defaultValue` clause didn't work (see [this commit](https://github.com/iron-fish/website-testnet/pull/226/commits/743d414298430af98b9c399534eac53d41128459) for what I tried). The core issue is whether `$touched` is set correctly -- if it isn't, the validation only happens after the user interacts with the field (which is the original issue Jason was seeing).

I'll investigate further (because this logic isn't ideal) but I'm inclined to largely keep things close to this commit as I think further refactoring will likely be a long process.